deeplake/core/transform/transform_dataset.py

```diff
@@ -1,40 +1,197 @@
-from deeplake.core.transform.transform_tensor import TransformTensor
 from deeplake.util.exceptions import TensorDoesNotExistError
+from deeplake.core.linked_sample import LinkedSample
+from deeplake.core.sample import Sample
+from deeplake.core.tensor import Tensor
+from typing import Union, List, Any
+from deeplake.constants import MB
+from itertools import chain
+
+import numpy as np
+
+import posixpath
+import bisect
+
+
+class TransformTensor:
```

Comment:
any reason why we shifted transform tensor to the same file?

Reply:
No reason, just thought it better to keep them together